ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-ref.yaml

```diff
@@ -1,6 +1,7 @@
 ref:
   as: baremetalds-devscripts-setup
   from: dev-scripts
+  timeout: 6h0m0s
```

Comment:
Is this really required? It could have impacts also on other non-agent jobs

Reply:
no, let's remove it.